Fix firehose documentation on es_datastream_name parameter (#3026)
elastic/observability-docs: docs/en/aws-firehose/aws-firehose-setup.asciidoc

*Parameters*:

> * Elastic recommends adding *`es_datastream_name`* parameter to help routing data to the correct integration data streams.
> If this parameter is NOT specified, data will be sent to the *`logs-generic-default`* data stream by default.
> *`es_datastream_name`* can be used to route documents to any data stream.
> Only when using Elastic integrations, setting the *`es_datastream_name`* parameter is required.

Reviewer: ```suggestion
* Elastic recommends setting the *`es_datastream_name`* parameter to help route data to the correct integration data streams.
If this parameter is not specified, data is sent to the *`logs-generic-default`* data stream by default.
* You can use the *`es_datastream_name`* parameter to route documents to any data stream.
When using Elastic integrations, you must set this parameter.
```

Author: Thank you!
